README.fr.md

```diff
@@ -0,0 +1,353 @@
+
+```
+mysql -u dvwa -p -D dvwa
+//puis tapez votre mot de passe dans le champ qui apparaît
+```
+
+Si le texte suivant apparaît (peut varier), les identifiants sont corrects:
+```
+Welcome to the MariaDB monitor.  Commands end with ; or \g.
+Your MariaDB connection id is 14
+Server version: 10.3.22-MariaDB-0ubuntu0.19.10.1 Ubuntu 19.10
+
+Copyright (c) 2000, 2018, Oracle, MariaDB Corporation Ab and others.
+
+Type 'help;' or '\h' for help. Type '\c' to clear the current input statement.
+
+MariaDB [dvwa]>
+```
+Puisque vous pouvez vous connecter en ligne de commande, il y a quelque chose qui cloche dans le fichier de configuration, re-vérifiez-le et signaler nus une erreur si vous n'arrivez pas à la trouver (sur github).
+
+Si le texte qui est apparu est le suivant, les identifiants sont incorrects. Répétez la [configuration de la base de données](#Paramétrage de la base de données) et soyez sûrs d'utiliser toujours les même identifiants durant la procédure.
+```
+ERROR 1045 (28000): Access denied for user 'dvwa'@'localhost' (using password: YES)
+//signifie: "Erreur 1045 (28000): Accès refusé pour l'utilisateur 'dvwa'@'localhost' (clé renseignée: OUI)"
+```
+
+Si vous obtenez l'erreur suivante, les identifiants sont corrects mais l'utilisateur n'a pas accès à la database.
+Réitérez aussi le paramétrage de la base de données et vérifiez le nom de la base de données.
+```
+ERROR 1044 (42000): Access denied for user 'dvwa'@'localhost' to database 'dvwa'
+//signifie: "Erreur 1044 (42000): Accès refusé pour l'utilisateur 'dvwa'@'localhost' à la BDD 'dvwa'."
+```
+
+La dèrnière erreur peut être celle-ci:
+```
+ERROR 2002 (HY000): Can't connect to local MySQL server through socket '/var/run/mysqld/mysqld.sock' (2)
+```
+Ça n'est pas une erreur d'authentification, c'est juste que le système de gestion de la BDD n'est pas activée. Démarrez-le en tapant :
+```sh
+sudo service mysql start
+```
+
+### Méthode d'identification inconnue
+
+Avec les versions les plus récentes de MySQL, PHP ne peut plus échanger avec la BDD dans sa configuration initiale. Si vous obtenez cette erreur, c'est que vous possédez cette configuration :(.
+```
+Database Error #2054: The server requested authentication method unknown to the client.
+//Signification: "Erreur de BDD  #2045: la méthode authentification utilisée est inconnue."
+```
+
+Vous avez deux options, la première étant de désinstaller MySQL et d'installez MariaDB. Ce lien vous envoie vers le manuel officiel de MariaDB.
+<https://mariadb.com/resources/blog/how-to-migrate-from-mysql-to-mariadb-on-linux-in-five-steps/> (en anglais)
+
+Sinon, suivez ces étapes:
+
+1. En tant que root, éditez le fichier `/etc/mysql/mysql.conf.d/mysqld.cnf`
+2. sous la ligne `[mysqld]`, ajoutez
```

Comment:
These should all be 1. so they render correctly and don't need changing if an item is added into the list.